Tasks/DownloadPackageV1/multifilepackage.ts

```diff
@@ -49,7 +49,8 @@ export class MultiFilePackage extends Package {
     private async getPackageFileContent(fileMetadata: any): Promise<Map<string, PackageFileResult>> {
         return new Promise<Map<string, PackageFileResult>>((resolve, reject) => {
             var resultMap = new Map<string, PackageFileResult>();
-            var content = fileMetadata.protocolMetadata.data.content as string;
+            let fileContent = fileMetadata.protocolMetadata.data.content || fileMetadata.protocolMetadata.data.Content;
```

Comment:
Do we need to check subproperties for null/undef? This code looks really fragile in terms of types :(

Reply:
Generally it would be great to refactor this code to add proper strict types instead of using `any` and typecasting potentially null/undef to `string`, but use your own judgement depending on how soon you need to get this fix and whether you have cycles to do it